addons/test_mail_full/controllers/portal.py

```diff
@@ -14,9 +14,21 @@ def test_portal_record_page(self, res_id, **kwargs):
         record = request.env["mail.test.portal"]._get_thread_with_access(res_id, **kwargs)
         values = {
             "object": record,
-            "token": kwargs.get("access_token", None),
-            "hash": kwargs.get("hash", None),
-            "pid": kwargs.get("pid", None),
+            "token": kwargs.get("token"),
+            "hash": kwargs.get("hash"),
+            "pid": kwargs.get("pid"),
```

Comment:
This change is not directly related to this PR, but I cleaned it up while I was at it.